src/fides/api/service/privacy_request/request_runner_service.py

```diff
@@ -288,15 +282,7 @@ def queue_privacy_request(
         from_webhook_id=from_webhook_id,
         from_step=from_step,
     )
-    try:
-        cache.set(
-            get_async_task_tracking_cache_key(privacy_request_id),
-            task.task_id,
-        )
-    except DataError:
-        logger.debug(
-            "Error tracking task_id for request with id {}", privacy_request_id
-        )
+    cache_task_tracking_key(privacy_request_id, task.task_id)
```

Comment:
Moved this to a method I can share when caching the celery task id for the Request Task